src/proof/proof_rule.h

```diff
@@ -23,11 +23,14 @@
 namespace cvc5::internal {
 
 /**
+ * \internal
+ * This documentation is target for the online documentation that can
+ * be found at https://cvc5.github.io/docs/cvc5-1.0.0/proofs/proof_rules.html.
```

Comment:
Actually, thinking about this, do we need to link with the version always?

Reply:
I'd suggest using https://cvc5.github.io/docs/latest/proofs/proof_rules.html

Reply:
OK, done